src/Config/appConfig.js

```diff
@@ -91,6 +91,9 @@ export const appLink = {
   manageTaskTemplateEdit: ({ teamId, taskId, version }) => `/manage/task-templates/team/${teamId}/${taskId}/${version}`,
   manageTaskTemplateYaml: ({ teamId, taskId, version }) =>
     `/manage/task-templates/team/${teamId}/${taskId}/${version}/yaml-editor`,
+
+  //external apps
+  docsWorkflowEditor: () => `${BASE_DOCUMENTATION_URL}/3.2.0/how-to-guide/workflow-editor`,
```

Comment:
Small change. Removed the `3.2.0` to make the value `${BASE_DOCUMENTATION_URL}/how-to-guide/workflow-editor`

This way it will always navigate to the latest version.